docs/modules/migrate/pages/lts.adoc

```diff
@@ -0,0 +1,130 @@
+= Long-term Support Releases
+
+:description: Hazelcast simplifies your upgrade experience with the introduction of long-term support (LTS) releases. You can upgrade directly from a supported previous release to the LTS release or directly between LTS releases using a rolling upgrade.
+:page-enterprise: true
+
+{description}
+
+LTS releases have been introduced with 5.5. You can upgrade directly from 5.2.x or later to 5.5 without the need to upgrade to each release between your current version and the LTS release. We recommend that you upgrade to 5.5 from the latest available patch for your current release.
+
+NOTE: Features marked as Beta in the source version are upgraded, but are supported only in a cluster of the source version due to changes that can occur between releases. For further information on rolling upgrade compatibilities, see xref:maintain-cluster:rolling-upgrades.adoc#hazelcast-members-compatibility-guarantees[Compatibility Guarantees].
+
+Between LTS releases, Hazelcast will provide short-term support (STS) releases. You can choose to upgrade directly from one LTS to the next LTS, or you can upgrade from one LTS to each STS release and then from the last STS release to the next LTS release. Before deciding the approach that best suits your needs, consider the following:
+
+[cols="1,1"]
+|===
+|LTS|STS
+
+|Focus on stability
+|Focus on innovation
+
+|Consolidates the functionality introduced in all STS releases since the last LTS release. Can include improvements, such as increased capacity limits
+|Provides the latest features and bug fixes
+
+|Released every two years
+|Released at least twice each year
+
+|Upgrade directly from the previous LTS, or from the STS release that precedes the latest LTS release
+|Upgrade from last available STS release only
+
+For example, if there are three STS releases between LTS releases, you cannot upgrade to the latest LTS unless you have upgraded to all three STS releases 
+
+|Full support for three years with the option of extended support for a further year
+|Maintenance support until the next STS or LTS release only
+
+|Choose if you value stability over new feature adoption, you want to upgrade less frequently, and you want full or extended support
+|Choose if you value new features over stability, you are willing to upgrade several times a year, and you are happy with maintenance support for the life of the STS
+|===
+
+== Hazelcast 5.5 LTS
+
+The Hazelcast 5.5 LTS release supports direct upgrade as follows:
+
+[cols="1,1"]
+|===
+|Upgrade From|LTS Release Support
+
+|5.2
+
+5.3
+
+a|* IMap data structure. This includes the API, configuration options, and persistence
+* CP subsystem. This inclues the API and peristence
+* WAN replication. Ongoing replication is supported during the upgrade. This includes both source and target; however, the source and the target cannot be upgraded simultaneously
+
+NOTE: Replication has been tested for 5.2 or later clusters only.
```

Comment:
But context for this line is only versions 5.2 and 5.3 anyway - does this add anything?

Reply:
Added from a comment from @frant-hartm - but a fair point.

What do you think, @frant-hartm, or did I miss the point of your original comment?

Reply:
When there is WAN replication you have 2 clusters and 3 different versions.

*from* version of cluster A (any of 5.2, 5.3, 5.4), 
*to* version of cluster A 

version of cluster B - this could be any Hazelcast version 4.0-5.4. 

We tested only 5.2, 5.3 and 5.4 for cluster B.

Reply:
Expanded to "NOTE: Hazelcast has tested WAN Replication to validate the upgrade solution. The test covered two clusters of different versions; for scenarios where WAN Replication is used from a cluster with a version or 5.2, 5.3, or 5.4 to another cluster with a version between 4.0 and 5.4, only the replication to clusters of version 5.2 or later have been tested."